src/ReactWrapper.js

```diff
@@ -288,6 +288,15 @@ export default class ReactWrapper {
   }
 
   /**
+   * Returns true if the component rendered nothing, i.e., null or undefined.
+   *
+   * @returns {boolean}
+   */
+  isNull() {
+    return this.single(n => findDOMNode(n) === null);
```

Comment:
That doesn't actually work. It also seems like SFC causes issues too. Trying to figure that out.